client/app_start.cpp

```diff
@@ -567,21 +567,21 @@ int ACTIVE_TASK::start(bool test) {
     if (app_version->avg_ncpus < 1) high_priority = true;
     if (app_version->is_wrapper) high_priority = true;
 
-    if (wup->project->verify_files_on_app_start) {
-        fip=0;
-        retval = gstate.input_files_available(result, true, &fip);
-        if (retval) {
-            if (fip) {
-                snprintf(
-                    buf, sizeof(buf),
-                    "Input file %s missing or invalid: %s",
-                    fip->name, boincerror(retval)
-                );
-            } else {
-                safe_strcpy(buf, "Input file missing or invalid");
-            }
-            goto error;
+    // make sure the task files exist
+    //
+    fip=0;
+    retval = gstate.task_files_present(result, true, &fip);
+    if (retval) {
+        if (fip) {
+            snprintf(
+                buf, sizeof(buf),
+                "Input file %s missing or invalid: %s",
```

Comment:
Since we're not verify the content of the file, this message is misleading. Should it be changed to:
`Input file %s is missing: %s` ?